sdk/textanalytics/Azure.AI.TextAnalytics/src/DocumentHealthcareResult.cs

```diff
@@ -0,0 +1,84 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+using System.Text.RegularExpressions;
+using Azure.AI.TextAnalytics.Models;
+
+namespace Azure.AI.TextAnalytics
+{
+    /// <summary>
+    /// DocumentHealthcareEntities.
+    /// </summary>
+    public partial class DocumentHealthcareResult
+    {
+        internal DocumentHealthcareResult(DocumentHealthcareEntitiesInternal documentHealthcareEntities)
+        {
+            Entities = documentHealthcareEntities.Entities;
+            Relations = documentHealthcareEntities.Relations != null ? ResolveHealthcareRelations(documentHealthcareEntities.Entities, documentHealthcareEntities.Relations) : null;
+            Id = documentHealthcareEntities.Id;
+            Warnings = documentHealthcareEntities.Warnings != null ? Transforms.ConvertToWarnings(documentHealthcareEntities.Warnings) : null;
```

Comment:
also, as FYI, in .NET we don't have `null` value for collections, we always assigned it so that customer will not get null reference exception

Reply:
This is something to check in the tests. i.e.
`Assert.IsNotNull(Warnings)`

Reply:
Should I assign it as an empty list?

Reply:
yes please.
also could you fix the swagger? i.e. https://github.com/Azure/azure-rest-api-specs/blob/master/specification/cognitiveservices/data-plane/TextAnalytics/preview/v3.1-preview.3/TextAnalytics.json#L1718